lapce-core/queries/protobuf/highlights.scm

```diff
@@ -0,0 +1,58 @@
+[
+  "syntax"
+  "package"
+  "option"
+  "import"
+  "service"
```

Comment:
Where are this files from?

Reply:
`folds.scm` is from the tree-sitter-protobuf repo and `highlights.scm` comes from helix-editor.

Reply:
could you please include that information just like other queries have?